lib/LanguageServerHover/Handler/HoverHandler.php

```diff
@@ -73,16 +73,19 @@ public function hover(
             $symbolContext = $offsetReflection->symbolContext();
             $info = $this->infoFromReflecionOffset($offsetReflection);
             $string = new MarkupContent('markdown', $info);
+            
 
             return new Hover($string, new Range(
-                PositionConverter::byteOffsetToPosition(
-                    ByteOffset::fromInt($symbolContext->symbol()->position()->start()),
-                    $document->__toString()
-                ),
-                PositionConverter::byteOffsetToPosition(
-                    ByteOffset::fromInt($symbolContext->symbol()->position()->end()),
-                    $document->__toString()
-                )
+                $position,
+                $position
```

Comment:
Not sure what to do about this one. It is really annoying but it probably belongs in its own PR.

Reply:
yes if not related please remove